plenario/api/shape.py

```diff
@@ -254,8 +253,25 @@ def export_shape(dataset_name):
     Expected query parameter: `data_type`. We expect it to be one of 'json', 'kml', or 'shapefile'.
                                 If none of these (or unspecified), return JSON.
     """
-    
-    return export_dataset_to_json_response(dataset_name)
+    '''
+    SELECT *
+    FROM {dataset_name} AS g
+    WHERE ST_Intersects(g.geom, ST_GeomFromGeoJSON('{geojson_fragment}'))
+    '''
+
+    params = request.args.copy()
+
+    columns_string = "*"
+    where_string = ""
+    #if 'columns' in params.keys() or 'bounding_box' in params.keys():
+    if params.get('columns'):
+        columns_string = params['columns']#",".join(params['columns'])
```

Comment:
Yeah, I think if someone did something like `columns='neighborhood;DROP TABLE some_important_data;SELECT neighborhood'` that would do the trick because the [ogr2ogr Postgres driver](http://www.gdal.org/drv_pg.html) passes SQL strings directly to postgres and then waits for the response.